new_platforms/src/optee_common.c

```diff
@@ -7,17 +7,274 @@
 #include <openenclave/bits/report.h>
 #include <openenclave/bits/result.h>
 
+#include <mbedtls/x509_crt.h>
+
+#include <RiotStatus.h>
+#include <RiotEcc.h>
+#include <RiotCrypt.h>
+#include <TcpsId.h>
+
 #ifndef _In_
 #include "sal_unsup.h"
 #endif
 
+unsigned char tcpsOid[] = {0x67, 0x81, 0x05, 0x05, 0x04, 0x02};
+
+// Adopted from mbedtls
```

Comment:
Does the fact that the GetClaimOidData is mostly taken from mbedtls affect the copyright header in any way?

Reply:
Any chance of sending a PR against mbed to expose this API?

Reply:
This API is not directly available out of mbed. It is based on a function in mbed, but customized to pick up our OID. While more generic version could be added into mbed, I feel like it is out of scope.

Reply:
I have moved it into mbedtls after all. I have not validated for any other oids, but that can be adjusted later.